package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -310,6 +325,8 @@ def __init__(self, universe,
         information is used, as this is the only way that guarantees the
         correct identification of donor-hydrogen pairs.
 
+        .. versionchanged:: 2.4.0
+            Added use of atom types
```

Comment:
I'd prefer indentation under "v" but given the precedence, I'm ok with indentation under "e".

I *think* we normally order the versionchanged as oldest to newest so then you'd need to switch order (but please check).

Reply:
You need *two* empty lines above versionchanged, otherwise formatting errors can happen.

This applies to _all_ versionchanged that you added.